src/ucs/debug/debug.c

```diff
@@ -1080,12 +1080,20 @@ void ucs_handle_error(const char *message)
     }
 }
 
+int ucs_debug_is_handle_errors()
+{
+    static const unsigned mask = UCS_BIT(UCS_HANDLE_ERROR_BACKTRACE) |
+                                 UCS_BIT(UCS_HANDLE_ERROR_FREEZE) |
+                                 UCS_BIT(UCS_HANDLE_ERROR_DEBUG);
+    return !!(ucs_global_opts.handle_errors & mask);
```

Comment:
why need `!!` here? Typically we do not do it for booleans, just for the cases where `0` or `1` is needed

Reply:
@brminich fixed